wampy/roles/publisher.py

```diff
@@ -29,7 +29,9 @@ def __call__(self, *unsupported_args, **kwargs):
                 "wampy requires at least one message to publish to a topic"
             )
 
-        message = Publish(topic=topic, options={}, **kwargs)
+        if "options" not in kwargs:
```

Comment:
i see what you're doing here. because of the way wampy publishes messages you are now forced to pass through the `options` dict through in the general `kwargs`, just as I was forced to pass through the `topic` URI in the `kwargs`. And I had `options` hard-coded as an empty dict... which is not particularly helpful.

Thanks for this @titilambert 
I'll add you to the CONTRIBUTORS.txt and get this released.